docs-source/content/faq/external-clients.md

```diff
@@ -1,57 +1,57 @@
+- Configure a custom channel for the T3 protocol in WebLogic that (A) enables HTTP tunneling, and (B) specifies an external address and port that correspond to the address and port remote clients will use to access the load balancer.  See  [Adding a WebLogic custom channel](#adding-a-weblogic-custom-channel) for samples and details.
 
-- Set up a load balancer that redirects HTTP traffic to the custom channel. For a discussion of load balancers, see [Ingress]({{<relref "/userguide/managing-domains/ingress/_index.md">}}). If you're also using OKE/OCI to host your Kubernetes cluster, also see [Using an OCI Load Balancer]({{<relref "/faq/oci-lb">}}).
+- Set up a load balancer that redirects HTTP traffic to the custom channel. For more information on load balancers, see [Ingress]({{<relref "/userguide/managing-domains/ingress/_index.md">}}). If you're also using OKE/OCI to host your Kubernetes cluster, also see [Using an OCI Load Balancer]({{<relref "/faq/oci-lb">}}).
 
 - __Important__: Ensure that the load balancer configures the HTTP flow to be 'sticky' - for example, a Traefik load balancer has a `sticky sessions` option. This ensures that all of the packets of a tunneling client connection flow to the same pod, otherwise the connection will stall when its packets are load balanced to a different pod.
 
 - Remote clients can then access the custom channel using an `http://` URL instead of a `t3://` URL.
 
-- Review the [Security Notes](#security-notes) below.
+- Review the [Security notes](#security-notes).
 
-#### Kubernetes NodePorts
+##### Kubernetes NodePorts
 
-The Kubernetes NodePorts approach for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster involves configuring a network channel on the desired WebLogic cluster that accepts T3 protocol traffic, and deploying Kubernetes NodePort that redirects external network traffic on the Kubernetes nodes to the network channel.
+The Kubernetes NodePorts approach for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster involves configuring a network channel on the desired WebLogic cluster that accepts T3 protocol traffic, and deploying a Kubernetes NodePort that redirects external network traffic on the Kubernetes nodes to the network channel.
 
-Here are the specific steps:
+Here are the steps:
 
-- Configure a custom channel for the T3 protocol in WebLogic that specifies an external address and port that are suitable for remote client use.  See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
+- Configure a custom channel for the T3 protocol in WebLogic that specifies an external address and port that are suitable for a remote client use.  See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
```

Comment:
'for a remote client use' sounds a little odd to me 'for remote client use' sounds better to me

Reply:
Yep, I screwed that one up. I'll fix it.